astroquery/vizier/tests/test_vizier.py

```diff
@@ -271,3 +275,19 @@ def test_get_catalog_metadata(self):
         assert "WHERE ivoid = 'ivo://cds.vizier/test'" in request_dict["QUERY"]
         with pytest.raises(ValueError, match="No catalog name was provided"):
             vizier.core.Vizier().get_catalog_metadata()
+
+    def test_get_catalog_metadata_empty_result(self, monkeypatch):
+        """Checks that an empty result raises a meaningful error."""
+        v = vizier.core.Vizier(catalog="should_return_empty_result")
+
+        def return_empty_votable(_):
+            if version.parse(astropy_version) < version.parse("6.0.0"):
```

Comment:
nitpicking, but I would suggest adding an ASTROPY_LT_6 to `utils/commons.py` as it will be easier to grep and clean up these once we drop support for astropy<6

Reply:
Done :)